### PR TITLE
[DNM] Issue #274 Check cookies first

### LINKED ISSUE #274: Check for cookies first and if no cookies then look for token_json

### PR DESCRIPTION
**Pull request by kishansagathiya**
While proxying using UI, first check if we find JSESSIONID cookie or
JenkinsIdled cookie and if we don't then look for token_json in the URL

Fixes #274

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @kishansagathiya for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by kishansagathiya** [comment]
Need to check if this is breaking anything

File: internal/proxy/ui_requests.go [hunk 1]
```diff
@@ -4,13 +4,14 @@ import (
...
 	"errors"
 	"fmt"
-	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/clients"
-	"github.com/satori/go.uuid"
-	log "github.com/sirupsen/logrus"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
+
```
Comment (sthaha) [inline internal/proxy/ui_requests.go]: does go fmt agree with this?

Comment (kishansagathiya) [reply internal/proxy/ui_requests.go]: That is done by go fmt. I didnt touch it.

**Pull request closed without merging**